cli/lib/kontena/cli/stacks/yaml/stack_file_loader.rb

```diff
@@ -27,7 +27,9 @@ def self.loaders
       # @return [StackFileLoader]
       def self.for(source, parent = nil)
         loader = loaders.find { |l| l.match?(source, parent) }
-        raise "Can't determine stack file origin for '#{source}'" if loader.nil?
+        if loader.nil?
+          raise RuntimeError, "Not found: no such file #{source} or invalid uri scheme"
```

Comment:
This slipped in from  #2809?

Reply:
Yes

Reply:
Huh, after merging that, the diff still shows up in the PR, without it being a conflict... let's see what happens.